pkg/permissions/verbs.go

```diff
@@ -18,36 +21,87 @@ const (
 	DELETE = Verb("DELETE")
 )
 
+var allVerbsOrder = []Verb{GET, POST, PUT, PATCH, DELETE}
+
 // VerbSet is a Set of Verbs
-type VerbSet []Verb
+type VerbSet map[Verb]bool
```

Comment:
An even more compact representation, since we known all the element that can compose the set, would be to use [a struct of 5 bools](http://golang-sizeof.tips/?t=Ly8gU2FtcGxlIGNvZGUKc3RydWN0IHsKCUdldCBib29sCglQb3N0IGJvb2wKCVB1dCBib29sCglQYXRjaCBib29sCglEZWxldGUgYm9vbAp9Cg==). You also would not have to worry about the order.